### PR TITLE
Footer design

### PR DESCRIPTION
**Pull request by PierreGoe**

### CODE REVIEW

**Review by LuigiMenez — DISMISSED** [review]
It's good for me

File: src/components/Footer/index.jsx [hunk 1]
```diff
@@ -1,25 +1,94 @@
...
+                <img className="linkSocial" src={linkedin} alt="logo github" />
+              </a>
+            </div>
+          </div>
+        </div>
+        <div className="member">
+          <div className="memberImage">
+            <img src={victore} alt="pictureOfMember" />
+          </div>
+          <div className="member-info">
```
Comment (Hacene010) [inline src/components/Footer/index.jsx]: Use camelCasse for all classNames to be coherent :)

File: src/components/Footer/index.jsx [hunk 2]
```diff
@@ -1,25 +1,94 @@
...
+            <h4>Victor</h4>
+            <span>Web Developer</span>
+            <div className="social-links">
```
Comment (Hacene010) [inline src/components/Footer/index.jsx]: same here

File: src/components/Footer/index.jsx [hunk 3]
```diff
@@ -1,25 +1,94 @@
...
+                <img className="linkSocial" src={linkedin} alt="logo github" />
+              </a>
+            </div>
+          </div>
+        </div>
+        <div className="member">
+          <div className="memberImage">
+            <img src={luigi} alt="pictureOfMember" />
+          </div>
+          <div className="member-info">
```
Comment (Hacene010) [inline src/components/Footer/index.jsx]: same here

File: src/components/Footer/index.jsx [hunk 4]
```diff
@@ -1,25 +1,94 @@
...
+            <h4>Luigi</h4>
+            <span>Web Developer</span>
+
+            <div className="social-links">
```
Comment (Hacene010) [inline src/components/Footer/index.jsx]: same

File: src/components/Footer/index.jsx [hunk 5]
```diff
@@ -1,25 +1,94 @@
...
+                <img className="linkSocial" src={linkedin} alt="logo github" />
+              </a>
+            </div>
+          </div>
+        </div>
+        <div className="member">
+          <div className="memberImage">
+            <img src={pierre} alt="pictureOfMember" />
+          </div>
+          <div className="member-info">
```
Comment (Hacene010) [inline src/components/Footer/index.jsx]: same

File: src/components/Footer/index.jsx [hunk 6]
```diff
@@ -1,25 +1,94 @@
...
+            <h4>Pierre</h4>
+            <span>Web Developer</span>
+            <div className="social-links">
```
Comment (Hacene010) [inline src/components/Footer/index.jsx]: same

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Seen some evils plz correct it

**Comment by LoicBrassart** [comment]
No SM validation

**Review by LuigiMenez — DISMISSED** [review]
It's good.

**Review by Hacene010 — DISMISSED** [review]
Ok for me

**Review by LuigiMenez — APPROVED** [review]
Fix conflict, linear-gradient in CocktaiCard style.css

**Review by Hacene010 — APPROVED** [review]
ok

**Pull request merged**